elements/glossy-portfolio-theme/lib/glossy-portfolio-breadcrumb.js

```diff
@@ -79,6 +84,12 @@ export class GlossyPortfolioBreadcrumb extends DDDSuper(I18NMixin(LitElement)) {
           margin-left: 0;
       
         }
+
+        @media (max-width: 575.98px) {
+          *{
```

Comment:
make it `:root` or `:host` if inside an element. the `*` is not recommended as it is set... EVERYWHERE that is found. CSS Vars will cascade through roots so can just set them globally and get same effect with less performance issues